packages/ts/src/components/crosshair/index.ts

```diff
@@ -179,6 +182,7 @@ export class Crosshair<Datum> extends XYComponentCore<Datum, CrosshairConfigInte
     const content = config.template(this.datum, this.xScale.invert(this.x))
     // Force set `followCursor` to `true` because we don't want Crosshair's tooltip to be hoverable
     tooltip.config.followCursor = true
+    tooltip.config.horizontalPlacement = tooltip.config.horizontalPlacement || Position.Right
```

Comment:
I think the original behavior will calculate the position based on the x position: https://github.com/f5/unovis/pull/402/files#diff-349d9b2f94ecdc1fe4615a4979bf0fee791c4006b0fed078fd416c31f90f0eb1L118

Reply:
@lee00678 Good point, I'll update the PR

Reply:
@lee00678 Fixed this problem but forgot about about one other thing. Will keep you posted